src/Glpi/Api/API.php

```diff
@@ -1879,13 +1879,22 @@ protected function createItems($itemtype, $params = [])
                     // to force having messages after redirect
                     $object["_add"] = true;
 
+                    $message = '';
+
                     //add current item
-                    $new_id = $item->add($object);
+                    try {
+                        $new_id = $item->add($object);
```

Comment:
```suggestion
                        $new_id = $item->add($object);
                        $message = $this->getGlpiLastMessage();
```
ANd shoudl be removed from the else